.flowconfig

```diff
@@ -27,5 +27,7 @@ suppress_type=$FlowSubtype
 esproposal.export_star_as=enable
 esproposal.optional_chaining=enable
 esproposal.nullish_coalescing=enable
-module.name_mapper='^@babel\/\([a-zA-Z0-9_\-]+\)$' -> '<PROJECT_ROOT>/packages/babel-\1/src/index'
+module.name_mapper='^@babel/\([a-zA-Z0-9_-]+\)$' -> '<PROJECT_ROOT>/packages/babel-\1/src/index'
+; Todo: can not make it works, 🤷‍♂️
+; module.name_mapper='^@babel/plugin-syntax-import-meta$' -> '<PROJECT_ROOT>/packages/babel-standalone/node_modules/@babel/plugin-syntax-import-meta/lib/index'
```

Comment:
Is this line commented out?

Reply:
Yes, I disabled the flow checks on generated plugins because I can't make it work. https://github.com/babel/babel/pull/12117/commits/41a2d6bdc722c204ff526220d95293eceb6309e3#diff-3dcf62f28000b8f05160cab1302d7a6cR24

I believe flow should respect `node_modules` symlinks and the `module.name_mapper` should not be specified at all.